vhdbuilder/packer/install-dependencies.sh

```diff
@@ -524,7 +524,21 @@ GPUContainerImages=$(jq  -c '.GPUContainerImages[]' $COMPONENTS_FILEPATH)
 
 NVIDIA_DRIVER_IMAGE=""
 NVIDIA_DRIVER_IMAGE_TAG=""
+NVIDIA_GRID_DRIVER_VERSION=""
 
+# Extract GRID driver version for release notes (applicable to all Linux distributions)
```

Comment:
wouldn't this logic be applicable for Ubuntu only ?  are we using the aks-gpu image for the grid driver installation on AzureLinux and other OS ?  This will also update the release notes for Flatcar/OsGuard I think..

Reply:
Yes, I just made the update as you'd commented